internal/artifacts/module_arch_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"archive/zip"
 	"errors"
 	"fmt"
+	"github.com/SAP/cloud-mta-build-tool/internal/platform"
```

Comment:
This import should not be in this import group

Reply:
moved